tools/eclipse/about.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Eclipse
+subsection: eclipse
+section: tools
+description: Full featured development environment for various programming languages integrating with plethora of different tools to make development process as integrated as possible
+---
+
+# What is Eclipse
+
+[Eclipse](https://www.eclipse.org/ide/) is a full featured, multi language IDE aiding developers in every step of software development lifecycle. Starting new project, working on established project with complex structure, monitoring and performance optimizing application, writing documentation, interacting with issue tracking systems or CI all from inside Eclipse IDE.
+
+## Installing Eclipse on Fedora
+
+If you are running Fedora Workstation it is strongly recommended to install Eclipse as shipped with Fedora as most of the times it contains important fixes, backported from upstream and not yet in released version:
```

Comment:
I would like to rephrase it like, I don't think, it is strongly recommended.
It depends on user.

What about something like that:
On Fedora install the eclipse package: